widgetssdk/src/main/java/com/glia/widgets/UiTheme.kt

```diff
@@ -1185,7 +1170,6 @@ data class UiTheme(
         }
 
         fun setTheme(theme: UiTheme) {
-            appBarTitle = theme.appBarTitle
```

Comment:
For my understanding, why don't we need it anymore?

Reply:
I have no clue how it happened, but this property wasn’t in use in the project and, therefore, had no effect.